packages/nouns-contracts/contracts/libs/Noracle.sol

```diff
@@ -115,6 +115,14 @@ library Noracle {
                 initializedObservationsFound++;
             }
         }
+
+        if (auctionCount > initializedObservationsFound) {
```

Comment:
general feedback: I feel like this function is a bit "fat". wonder if we can trim it.

is the `if (obs.initialized)` actually needed?
in which case is there an uninitialized member? wouldn't we need to break in that case?

Reply:
you're absolutely right, my bad.
the case of encountering uninitialized items is when we increase cardinality, and as we're iterating backwards, we loop back to the end where the new cardinality items are all in that state.

Reply:
did some cleanup here, lmk wdyt: https://github.com/nounsDAO/nouns-monorepo/pull/598/commits/b8557264a97216dfe1bb35a9c8705a0439fe35eb